qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -20,15 +20,14 @@
 from qiskit.qasm import pi
 from qiskit.circuit import EquivalenceLibrary, Parameter, QuantumCircuit, QuantumRegister
 
+from qiskit.quantum_info.synthesis.ion_decompose import cnot_rxx_decompose
```

Comment:
i think this file hasn't been added

Reply:
This file should already be on master, https://github.com/Qiskit/qiskit-terra/blob/4bd91fe/qiskit/quantum_info/synthesis/ion_decompose.py . (It's the same import from https://github.com/Qiskit/qiskit-terra/blob/4bd91fe/qiskit/quantum_info/synthesis/ion_decompose.py )